Source/WebCore/html/MediaElementSession.cpp

```diff
@@ -1118,10 +1118,58 @@ bool MediaElementSession::allowsPlaybackControlsForAutoplayingAudio() const
 }
 
 #if ENABLE(MEDIA_SESSION)
+static bool isDocumentPlayingSeveralMediaStreams(Document& document)
+{
+    size_t activeMediaElementsWithMediaStream = 0;
+    document.forEachMediaElement([&](auto& element) {
+        if (element.hasMediaStreamSrcObject() && element.isPlaying())
+            activeMediaElementsWithMediaStream++;
+    });
+
+    return activeMediaElementsWithMediaStream > 1 && MediaProducer::isCapturing(document.mediaState());
+}
+
+static bool processRemoteControlCommandByDocument(Document& document, PlatformMediaSession::RemoteControlCommandType commandType)
+{
+    auto* page = document.page();
+    if (!page)
+        return false;
+
+    if (!isDocumentPlayingSeveralMediaStreams(document))
+        return false;
```

Comment:
Because this function does nothing unless there are active media streams, I think something like `processRemoteControlCommandIfPlayingMediaStreams` might be easier to understand.

Reply:
Done